### DISCUSSION TITLE
Want to know if linking url to any react-konva layer like Rect/Text can be done?

### DISCUSSION THREAD

**Discussion by ahmadrazach** [konvajs/konva#1493] [Q&A] [answered]
Hy everyone, so I am making an editor from where everyone can make stories just like we make on Instagram. I am in more than a half way and I am loving the process of development.

 I want to know if we can attach any URL link to the konva Layer just like any link attached to the Instagram story.
 
For Example, if I add a Rect or Text layer and if user clicks it, it would take it to a specific URL.

So is that possible with konva JS? Again , I am developing using react-konva.

**Comment by lavrton** [comment, marked as answer]
There is no such thing as "link" in react-konva. It is just a canvas element where you draw. 
But what you can do is:

1. Listen to click event on any node (stage, layer, group, shape)
2. On click, manually open url on new tab

```js
<Rect
  {...props}
  onClick={() => {
    window.open(url, '_blank');
  }}
/>
```

**Reply by ahmadrazach** [reply]
Thanks for answering that
